src/screens/HyperSwitch/UserManagement/InviteUsers.res

```diff
@@ -112,8 +130,21 @@ let make = () => {
         ]
         ->Dict.fromArray
         ->Js.Json.object_
-      let _ = inviteUserReq(body, index)
+      let _ = inviteUserReq(body, index, emailPasswordsArray)
```

Comment:
Can we directly pass empty here in the same line?

Reply:
We need array reference name as this variable being used later.